sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/BearerTokenAuthenticationPolicy.java

```diff
@@ -63,46 +66,25 @@ public BearerTokenAuthenticationPolicy(TokenCredential credential, String... sco
     /**
      * Synchronously executed before sending the initial request and authenticates the request.
      *
-     * @param httpRequest The request context.
+     * @param httpRequest The request.
      */
-    public void authorizeRequestSync(HttpRequest httpRequest) {
-        setAuthorizationHeaderHelperSync(httpRequest, new TokenRequestContext().addScopes(scopes), false);
+    public void authorizeRequest(HttpRequest httpRequest) {
+        setAuthorizationHeaderHelper(httpRequest, new TokenRequestContext().addScopes(scopes), false);
     }
 
     /**
      * Handles the authentication challenge in the event a 401 response with a WWW-Authenticate authentication challenge
      * header is received after the initial request and returns appropriate {@link TokenRequestContext} to be used for
      * re-authentication.
      *
-     * @param httpRequest The request context.
+     * @param httpRequest The request.
      * @param response The Http Response containing the authentication challenge header.
-     *
      * @return A boolean indicating if containing the {@link TokenRequestContext} for re-authentication
      */
-    public boolean authorizeRequestOnChallengeSync(HttpRequest httpRequest, Response<?> response) {
+    public boolean authorizeRequestOnChallenge(HttpRequest httpRequest, Response<?> response) {
```

Comment:
Are we happy with the design of these APIs? Any regrets we want to fix now while we can still change it.

Reply:
Yeah fine as of current shape, if any changes needed to support any dowstream features, will revisit.